Cubical/Foundations/Path.agda

```diff
@@ -351,3 +352,103 @@ compPathR→PathP∙∙ {p = p} {q = q} {r = r} {s = s} P j i =
+  Iso.fun rotateSquareIso = rotateSquare
+  Iso.inv rotateSquareIso = rotateSquare⁻
+  Iso.rightInv rotateSquareIso _ = refl
+  Iso.leftInv rotateSquareIso _ = refl
+
+
+  rotateSquare≃ : ◰ ≃ ◳
+  rotateSquare≃ = isoToEquiv rotateSquareIso
+
+
+  squareTwisting : (x : ◰) → PathP (λ i → ≡rotateSquare i) x (rotateSquare x)
+  squareTwisting  x k i j =
+         hcomp
+            (λ l → λ {
+                    (k = i0) → x (i ∧ l) (j ∧ l)
+                  ; (k = i1) → x (~ j ∧ l) (i ∧ l)
+                  ; (i = i0) → hcomp
+                                (λ w →
+                                  λ { (l = i0) → x i0 i0
+                                    ; (k = i0) → x i0 (w ∧ j ∧ l)
+                                    ; (k = i1) → x (w ∧ ~ j ∧ l) i0
+                                    ; (j = i0) → x (w ∧ l ∧ k) i0
+                                    ; (j = i1) → x i0 (w ∧ l ∧ (~ k))
+                                    })
+                                (x i0 i0)
+                  ; (i = i1) → hcomp
+                                (λ w →
+                                  λ { (l = i0) → x i0 i0
+                                    ; (k = i0) → x l (l ∧ (j ∨ ~ w))
+                                    ; (k = i1) → x ((~ w ∨ ~ j) ∧ l) l
+                                    ; (j = i0) → x l ( (k ∨ ~ w) ∧ l)
+                                    ; (j = i1) → x ((~ k ∨ ~ w) ∧ l) l
+                                    })
+                                (x l l)
+                  ; (j = i0) → hcomp
+                                (λ w →
+                                  λ { (l = i0) → x (~ w) i0
+                                    ; (k = i0) → x ((i ∧ l) ∨ ~ w) i0
+                                    ; (k = i1) → x (~ w ∨ l) (i ∧ l ∧ w)
+                                    ; (i = i0) → x ((~ w) ∨ (l ∧ k)) i0
+                                    ; (i = i1) → x (l ∨ ~ w) (k ∧ w ∧ l)
+                                    })
+                                (x i1 i0)
+                  ; (j = i1) → hcomp
+                                (λ w →
+                                  λ { (l = i0) → x i0 (~ w)
+                                    ; (k = i0) → x ( i ∧ l ∧ w) (l ∨ ~ w)
+                                    ; (k = i1) → x i0 ((i ∧ l) ∨ ~ w)
+                                    ; (i = i0) → x i0 ((l ∧ ~ k) ∨ ~ w)
+                                    ; (i = i1) → x ((~ k) ∧ l ∧ w) (l ∨ ~ w)
+                                    })
+                                (x i0 i1)
+                   })
+            (x i0 i0)
+
+
+
+  rotateSquare≃-≡rotateSquare-lemma :
+        rotateSquare≃
+          ≡
+        pathToEquiv ≡rotateSquare
+  rotateSquare≃-≡rotateSquare-lemma = equivEq (funExt p)
+    where
+
+      p : (x : Square _ _ _ _) → rotateSquare x ≡ transport ≡rotateSquare x
+      p x k i j =
+         hcomp
+           (λ l →
+             primPOr k (~ k ∨ i ∨ ~ i ∨ j ∨ ~ j)
+              (λ _ → coe0→i (λ i' → ≡rotateSquare i') l x i j)
```

Comment:
Can't you just use a transpFill here and get rid of the dependency on CartesianKanOps?